network/p2p/gossip/gossip_test.go

```diff
@@ -123,6 +131,12 @@ func TestGossiperGossip(t *testing.T) {
 
 			metrics, err := NewMetrics(prometheus.NewRegistry(), "")
 			require.NoError(err)
+
+			testHistogram := &testHistogram{
+				Histogram: metrics.bloomFilterHitRate,
+			}
+			metrics.bloomFilterHitRate = testHistogram
+
```

Comment:
This looks a bit hacky to me, if we want to do an assert that the metrics are the values we're expecting would it make sense for us to consider using [prometheus/testutil](https://pkg.go.dev/github.com/prometheus/client_golang/prometheus/testutil) here?